src/framing/Frame.cpp

```diff
@@ -95,78 +95,120 @@ std::ostream& operator<<(std::ostream& os, ErrorCode errorCode) {
   return os << "ErrorCode(" << static_cast<uint32_t>(errorCode) << ")";
 }
 
-std::ostream& operator<<(std::ostream& os, FrameFlags frameFlags) {
-  // TODO Match the Flag names with the AllowedFlags in the Frame declarations
+constexpr const char sFrameFlags_EMPTY[] = "0x00";
+constexpr const char sFrameFlags_METADATA[] = "METADATA";
+constexpr const char sFrameFlags_RESUME_ENABLE[] = "RESUME_ENABLE";
+constexpr const char sFrameFlags_LEASE[] = "LEASE";
+constexpr const char sFrameFlags_KEEPALIVE_RESPOND[] = "KEEPALIVE_RESPOND";
+constexpr const char sFrameFlags_FOLLOWS[] = "FOLLOWS";
+constexpr const char sFrameFlags_COMPLETE[] = "COMPLETE";
+constexpr const char sFrameFlags_NEXT[] = "NEXT";
```

Comment:
I wouldn't name these `sFrameFlags_NEXT`.  The convention for string literals is `kStringLiteralName`, and I think we can get away with doing `kNext` instead of duplicating the `sFrameFlags` prefix everywhere.